.expeditor/verify.pipeline.yml

```diff
@@ -616,5 +611,3 @@ steps:
         privileged: true
         single-use: true
         shell: ["powershell", "-Command"]
-        environment:
-          - HAB_NONINTERACTIVE=true
```

Comment:
This didn't work, not sure why. Moved inside the bash scripts run by hab instead.